reason-react-native/src/apis/Style.re

```diff
@@ -1,5 +1,28 @@
 type t;
 
+external array: array(t) => t = "%identity";
+external arrayOption: array(option(t)) => t = "%identity";
+/* list works too since RN accept recursive array of styles (ocaml list are recursive arrays in JS)*/
+external list: list(t) => t = "%identity";
+external listOption: list(option(t)) => t = "%identity";
+
+// Escape hatch, in case something is added into RN but unsupported,
+// Useful if you play with fancy platforms
+// Use with caution
+let unsafeAddProp: (t, string, string) => t =
+  (style, property, value) => {
+    Js.Obj.empty()
+    ->Js.Obj.assign(style->Obj.magic)
+    ->Js.Obj.assign(
+        {
+          let o = Js.Dict.empty();
+          o->Js.Dict.set(property, value);
+          o->Obj.magic;
+        },
+      )
+    ->Obj.magic;
+  };
+
```

Comment:
This is basically the same technique as in reason-react codebase. Not zero-cost but we can't do magic here :/